docs/articles/deployment/windows-service.md

```diff
@@ -0,0 +1,188 @@
+    })
+    .Build();
+
+await host.RunAsync();
+```
+
+### AkkaService.cs
+
+```csharp
+public sealed class AkkaService : BackgroundService
+    {
+        private readonly ILogger<AkkaService> _logger;
+        private readonly JokeService _jokeService;
+        private readonly ActorSystem _actorSystem;
+        private readonly IActorRef _actorRef;   
+
+        public AkkaService(JokeService jokeService, ILogger<AkkaService> logger, IServiceProvider serviceProvider)
+        {
+            _jokeService = jokeService; 
+            _logger = logger;
+            var bootstrap = BootstrapSetup.Create();
+
+
+            // enable DI support inside this ActorSystem, if needed
+            var diSetup = DependencyResolverSetup.Create(serviceProvider);
+
+            // merge this setup (and any others) together into ActorSystemSetup
+            var actorSystemSetup = bootstrap.And(diSetup);
+
+            // start ActorSystem
+            _actorSystem = ActorSystem.Create("akka-system", actorSystemSetup);
+            _actorRef = _actorSystem.ActorOf(MyActor.Prop());
+        }
+        protected override async Task ExecuteAsync(CancellationToken stoppingToken)
+        {
+            while (!stoppingToken.IsCancellationRequested)
+            {
+                string joke = await _jokeService.GetJokeAsync();
+                _actorRef.Tell(joke);   
+                _logger.LogWarning(joke);
+
+                await Task.Delay(TimeSpan.FromMinutes(1), stoppingToken);
+            }
+        }
+    }
+```
+
+### JokeService.cs
+
+```csharp
+public class JokeService
+    {
+        private readonly HttpClient _httpClient;
+        private readonly JsonSerializerOptions _options = new()
+        {
+            PropertyNameCaseInsensitive = true
+        };
+
+        private const string JokeApiUrl =
+            "https://karljoke.herokuapp.com/jokes/programming/random";
+
+        public JokeService(HttpClient httpClient) => _httpClient = httpClient;
+
+        public async Task<string> GetJokeAsync()
+        {
+            try
+            {
+                // The API returns an array with a single entry.
+                Joke[]? jokes = await _httpClient.GetFromJsonAsync<Joke[]>(
+                    JokeApiUrl, _options);
+
+                Joke? joke = jokes?[0];
+
+                return joke is not null
+                    ? $"{joke.Setup}{Environment.NewLine}{joke.Punchline}"
+                    : "No joke here...";
+            }
+            catch (Exception ex)
+            {
+                return $"That's not funny! {ex}";
+            }
+        }
+    }
+
+    public record Joke(int Id, string Type, string Setup, string Punchline);
+```
+
+This is of course, one way of deploying Akka.NET HEADLESS Service. Our blog post [How to Build Headless Akka.NET Services with IHostedService](https://petabridge.com/blog/akkadotnet-ihostedservice/) is a good alternative!
+
+## Azure PaaS Worker Role
```

Comment:
Move this to its own article - "Azure Cloud Service (Deprecated)" should be the title.

Mention in a DocFx !important comment that cloud services are largely deprecated in Azure now.